modules/apigee_edge_teams/css/apigee_edge_teams.switcher.css

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * version 2 as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301, USA.
+ */
+
+.team-switcher .dropbutton .dropbutton-action > * {
+  padding: 0.25em 1em;
+}
+
+.team-switcher .dropbutton .dropbutton-action span {
+  display: block;
+}
+
+.team-switcher .dropbutton-wrapper .dropbutton-widget {
+  text-align: left;
+}
+
+/* Style the dropbutton-no-action as a dropbutton divider. */
+.team-switcher .dropbutton-action > span.dropbutton-no-action {
```

Comment:
> I would suggest to implement a simple and stable solution in the module and make it fancy in the Kickstart theme. Because the look and feel that @Jaesin shared from Drupal 7 is typically something that should not be implemented in a module.
https://github.com/apigee/apigee-edge-drupal/pull/146#issuecomment-470956425
> Therefore we tried to use only Drupal 8 core render elements wherever it was possible because that way we do not need to worry about how they look with different themes.

If you start adding custom styles to Drupal core form elements, you should start worrying about again how these tweaks look like in different themes. Can't we avoid this in the module and use the dropdown render element as it looks like by default? (Make it fancy in the Kickstart theme as I suggested earlier.)

@Jaesin @cnovak

Reply:
I don't see any issue with this CSS since it's prefixed with a class and wouldn't affect anything except the switcher. I would prefer to have the component provider present the component in a sane way. If someone installs the module without the kickstart, the feature should be usable as it was installer. It shouldn't require a custom theme to make it usable.

Reply:
Are you saying that dropdown render element without CSS is not usable? Do you have a screenshot about how it looks like now?

Reply:
@mxr576 the first element from the dropbutton is a non-link. So we added minimum styling to remove the hover effect from the first element. That's it.

Reply:
<img width="939" alt="image" src="https://user-images.githubusercontent.com/159693/54638827-1c565a00-4a49-11e9-92b0-582f7c9264bd.png">